duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -323,6 +343,7 @@ public static class HttpBuilder {
 
     private int timeout = DEFAULT_TIMEOUT_SECS;
     private String[] caCerts = null;
+    private Map<String, String> additionalDuoHeaders = new TreeMap<String, String>();
```

Comment:
let's make the left side `SortedMap<String, String>` so future me remembers it's always sorted